components/mpas-seaice/bld/build-namelist

```diff
@@ -416,6 +416,7 @@ my $RUN_STARTDATE          = "$xmlvars{'RUN_STARTDATE'}";
 my $START_TOD              = "$xmlvars{'START_TOD'}";
 my $RUN_REFDATE            = "$xmlvars{'RUN_REFDATE'}";
 my $CONTINUE_RUN           = "$xmlvars{'CONTINUE_RUN'}";
+my $wave_spec              = $xmlvars{'WAV_SPEC'};
```

Comment:
you may need quotes around $xmlvars{'WAV_SPEC'}, as in the lines before this

Reply:
the quotes do not seem to matter -this part works fine as is - but I can add them for consistency.